pkg/router/f5/plugin.go

```diff
@@ -510,6 +510,28 @@ func (p *F5Plugin) deleteRoute(routename string) error {
 				return err
 			}
 		}
+	} else {
```

Comment:
I just saw other three type routes are checked in sequence in the function. Seems only one type route will be matched for a given routename.
LGTM. Thank you.